client/my-sites/stats/mini-carousel/index.jsx

```diff
@@ -79,6 +79,7 @@ const MiniCarousel = ( { slug, isSitePrivate } ) => {
 
 	const showGoogleAnalyticsPromo =
 		! useSelector( isBlockDismissed( EVENT_GOOGLE_ANALYTICS_BANNER_DISMISS ) ) &&
+		! jetpackNonAtomic &&
 		( isFreePlan( currentPlanSlug ) || isPersonalPlan( currentPlanSlug ) );
```

Comment:
Should this `isPersonalPlan` be updated to check also new plan names or this works as-is?